src/client/sagas/fetch/index.js

```diff
@@ -13,7 +14,7 @@ export function* fetchSaga(
   try {
     yield put(actions.clearError(actionName))
     yield put(actions.sendingRequest(actionName, true))
-    return yield call(get, graphQuery, variables)
```

Comment:
@kkateq why are we changing the get with post?. Do we have separate get vs post request?